api/src/backend/api/models.py

```diff
@@ -615,6 +615,7 @@ class DeltaChoices(models.TextChoices):
     id = models.UUIDField(primary_key=True, default=uuid7, editable=False)
     inserted_at = models.DateTimeField(auto_now_add=True, editable=False)
     updated_at = models.DateTimeField(auto_now=True, editable=False)
+    first_seen_at = models.DateTimeField(editable=False)
```

Comment:
What happens when we perform this migration on an already existing database with findings? What value do the older findings take? I believe we are missing, at least, `null=True` here and in the migration.

Reply:
> Sorry, this comment was meant to be added with the last review but I missed it somehow.